### DISCUSSION TITLE
Corner Radius only in te bottom?

### DISCUSSION THREAD

**Discussion by RickWalkerOne** [TomSchimansky/CustomTkinter#1494] [General]
I want to know how I can create in CTkCanvas and DrawEngine a Canvas (Background) with only a side with round. For exemple, on the css we can configure border radios on top_left, botton_right, top_right and botton_left. I can do anything like this?

Because I want to create a table on CustomTkinter (like a datagridview of C#), but I want to made on CustomTkinter theme. I.e.: first row and first column with a background darkest.

**Comment by Akascape** [comment]
@RickWalkerOne You can use `background_corner_color=["color1", "color2", "color3", "color4"]` in ctkframe or ctkbutton.

>Because I want to create a table on CustomTkinter...

Here is a readymade simple table widget you can use in customtkinter: https://github.com/TomSchimansky/CustomTkinter/discussions/1483
